### ISSUE TITLE
Too few kill feed lines

### ISSUE THREAD

**Issue by randrade86** [CriticalFlaw/FlawHUD#113]
How about increase the amount of lines that display the kill feed? 4 lines is way too small.

**Comment by CriticalFlaw** [comment]
I think anything beyond 5-6 will be too much and take up too much space. The users are also able to expand it themselves in scripts/hudlayout.res

**Comment by h9lk** [comment]
4 just right, more - distracting & open space for spam, for example look at spam demo with crits in kill feed, why... Very uncomfortable for me

**Comment by randrade86** [comment]
> 4 just right, more - distracting & open space for spam, for example look at spam demo with crits in kill feed, why... Very uncomfortable for me
I respect your opinion but for me 4 = too few, you can't even figure out what is going on the map (I'm not talking about ultiduos, 4x4 or 6x6). Not sure what resolution you are playing, but I modified the `scripts/hudlayout.res` and looks like 8 lines are great on full HD.

I have been playing TF2 since 2008, with over 5200h of gameplay and I strongly believe that the kill feed + scoreboard are great tools for awareness.

**Comment by CriticalFlaw** [comment]
I don't have an issue with bumping it up to five or six because servers with 18 to 32 players are common and, especially on KOTH maps, skirmishes will have more than four killfeed events occurring in a short period of time.

**Comment by h9lk** [comment]
> 
> 
> > 4 just right, more - distracting & open space for spam, for example look at spam demo with crits in kill feed, why... Very uncomfortable for me
> 
> I respect your opinion but for me 4 = too few, you can't even figure out what is going on the map (I'm not talking about ultiduos, 4x4 or 6x6). Not sure what resolution you are playing, but I modified the `scripts/hudlayout.res` and looks like 8 lines are great on full HD.
> 
> I have been playing TF2 since 2008, with over 5200h of gameplay and I strongly believe that the kill feed + scoreboard are great tools for awareness.

forgot that someone else is playing casual or community servers, use 4:3 1024x768, first time hear about using killfeed as a source of information on a map maybe for hl it's used, for me everything is determined by gamesense, maincall not done from kill feed for sure, most important thing in game is information about enemy uber and your + positions and damage, who was killed can be seen by clicking Tab

**Comment by randrade86** [comment]
> Agree with your position regarding on FullHD, it's more a matter of taste and habits, respect your experience of the game but it seems to me that not everyone will be comfortable as well as you with 5+ lines

That's why it would be nice to have the kill feed line amount as a configurable setting in Flawhud installer.

**Comment by CriticalFlaw** [comment]
> > Agree with your position regarding on FullHD, it's more a matter of taste and habits, respect your experience of the game but it seems to me that not everyone will be comfortable as well as you with 5+ lines
> 
> That's why it would be nice to have the kill feed line amount as a configurable setting in Flawhud installer. 

Good idea. I'll bump up the killfeed to 6 (nice even number) and transfer this ticket to the Installer repo to be added in as an option.
